include/nbl/core/containers/CMemoryPool.h

```diff
@@ -80,6 +80,12 @@ class NBL_API CMemoryPool : public Uncopyable
         return free_n<T>(ptr, 1u);
     }
 
+    // TODO are we gonna keep this?
+    void reset()
+    {
+        m_alctr.reset();
+    }
+
```

Comment:
yes but check/test `SimpleBlockBasedAllocator::reset` actually works

Reply:
Is this still relevant?

Reply:
well you still need to check the `CMemoryPool` reset works, but obviously it wont call the dtors of anything you allocated over it.